src/components/App.js

```diff
@@ -47,7 +46,7 @@ class App extends Component<{}, State> {
 
     try {
       newData = JSON.parse(newData)
-      newData = Object.values(newData)
+      newData = _.toArray(newData)
```

Comment:
Any reason to use `_.toArray()` rather than `Object.values()` here please? Also, why not storing the data as an array directly if that's how it will be hydrated?

Reply:
Yeah, this was actually something I meant to ask about but forgot. When I use Object.values() I get a flow error (https://github.com/facebook/flow/issues/2133) and I wasn't sure if it was preferable to keep it simple by using lodash or if I should use a workaround, like using Object.keys() (more expensive) or some custom flow function to deal with this case?

Reply:
That being said.. It is unnecessary here, I was writing to local storage as an array, so I'll remove this line & push. But I am still using lodash in state init in App.js, so let me know your preference and I'm happy to change that to whichever way is preferable.